### PR TITLE
Feat/collections

### LINKED ISSUE #2585: Explorer: Add First-time grantees & Grants Stack Veterans Collections to Explorer
**Overview**

Add the following Collections

### First Time Grantees 
Definition: projects accepted into GG19 that have no past applications 

**Title**: First Time Grantees 
**Byline**: Gitcoin 
**Description**: This collection showcases all grantees in GG19 that have not participated in a past round on Grants Stack! Give these first-time grantees some love (and maybe some donations, too!) 

### Grants Stack Veterans 
Definition: projects accepted into GG19 that were also accepted into at least one past Gitcoin round (GG18 or Beta rounds) 
- For a list of contract addresses for rounds in GG18 and Beta Rounds, see [here](https://docs.google.com/spreadsheets/d/1EIvwpxs4TjRtruavAb7CH1ogBHv0LLv-pXVhn2HSfpE/edit#gid=0) 

**Title**: Grants Stack Veterans 
**Byline**: Gitcoin 
**Description**: This collection showcases all grantees in GG19 that have participated in a past GG18 and/or Beta Round! Give these Grants Stack Veterans some love (and maybe some donations, too!) 

**Details**
Say more

### PR DESCRIPTION
**Pull request by carlbarrdahl**
- Fixes #2585 

Todo:
- [x] Add all projects in collection to cart
- [x] Share collection (copy url)

## PR checklist

For every PR, make sure that these statements are true:
- [ ] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [ ] Doesn't contain type casts and non-null assertions.
- [ ] Doesn't add `@ts-ignore`.
- [ ] Doesn't disable lints.
- [ ] Doesn't use `useState` just for computation - use plain variables instead.
- [ ] Splits components into pure components that don't depend on external state or hooks.
- [ ] Avoid embedding components within other components
- [ ] Doesn't propagate optional values without good reason, doesn't mark property values as optional if that doesn't represent reality.
- [ ] Doesn't duplicate existing code.
- [ ] Parses out-of-domain data - this includes user input, API respones, on-chain data etc.
- [ ] Doesn't contain commented out code.
- [ ] Doesn't contain skipped or empty tests.
- [ ] If this PR adds/updates any feature, it adds/updates its test script

Subjective - at the discretion of the reviewers
- Does things as simply as possible, but not simpler.
- Doesn't reinvent the wheel or create premature abstractions.

##### Refers/Fixes

fixes #issuenumber

### CODE REVIEW

File: packages/grant-explorer/src/features/collections/CollectionDetails.tsx [hunk 1]
```diff
@@ -22,7 +29,40 @@ export function CollectionDetails({ collection }: Props) {
         </Skeleton>
       </div>
 
-      <div className="text-lg">{collection.description}</div>
+      <div className="flex">
+        <div className="text-lg flex-1">{collection.description}</div>
+        <div className="w-96">
+          <div className="flex justify-end gap-2">
+            <Button variant="primary" onClick={() => alert("not implemented")}>
```
Comment (vacekj) [inline packages/grant-explorer/src/features/collections/CollectionDetails.tsx:36]: Is this supposed to not be implemented?

**Comment by melissa-neira** [comment]
hey @carlbarrdahl, it's been decided that there will be 10 collections in total, the design has changed slightly. you can view it [here](https://www.figma.com/file/1uxgU2jb0yyN11SUgxARmF/Explorer-S19?type=design&node-id=1613-18772&mode=design&t=omK69IuCM4BtOWno-4)

File: packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx [hunk 2]
```diff
@@ -19,7 +18,7 @@ import { CollectionDetails } from "../collections/CollectionDetails";
 import { FilterDropdown, FilterDropdownOption } from "../common/FilterDropdown";
 import { allChains } from "../../app/chainConfig";
 
-type Filter =
+export type Filter =
```
Comment (boudra) [inline packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx:21]: this filter is defined twice, once here and in `useApplications`, are they the same thing?

Comment (carlbarrdahl) [reply packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx:21]: Thanks. Updated!

File: packages/grant-explorer/src/features/collections/CollectionDetails.tsx (hunk 1 above)
Comment (carlbarrdahl) [reply packages/grant-explorer/src/features/collections/CollectionDetails.tsx]: Fixed!

**Review by michellema1208 — COMMENTED** [review]
@carlbarrdahl ty for such a quick turnaround!! Noticing two small things: 

1. This is my fault / typo -- for the Grants Stack Veterans Collection, in the description, Veteran should be plural / say "Veterans" 

2. I'm noticing on the preview card that there are 401 projects in each of the Collections, but when I open the CSVs from @boudra , it looks like there are 313 first-time grantees and 252 veterans. Please keep me honest, though!

**Comment by melissa-neira** [comment]
+1 to @michellema1208's comments

**Review by melissa-neira — APPROVED** [review]
lgtm design-wise!

**Review by michellema1208 — APPROVED** [review]
LGTM, amazing work as always Carl!

**Pull request merged**